build/commands/lib/config.js

```diff
@@ -40,7 +40,7 @@ var packageConfig = function (key, sourceDir = rootDir) {
   // packages.config should include version string.
   let obj = Object.assign({}, packages.config, { version: packages.version })
   for (var i = 0, len = key.length; i < len; i++) {
-    if (obj === undefined) {
+    if (!obj) {
```

Comment:
Is `obj` == to either `""` or `0` not a valid object?

Reply:
i haven't updated this change in the PR. it's happening probably because of cpp lint formatting.